cursorless-talon/src/cheatsheet/sections/modifiers.py

```diff
@@ -45,157 +50,238 @@ def get_modifiers():
+                "variations": [
                     {
-                        "spokenForm": f"{complex_modifiers['first']} <number> <scope>s",
-                        "description": "first <number> instances of <scope> in iteration scope",
+                        "spokenForm": f"{complex_modifiers['every']} <scope>",
+                        "description": "Every instance of <scope>",
                     },
+                ],
+            }
+        )
+
+    if "ancestor" in complex_modifiers:
+        items.append(
+            {
+                "id": "ancestor",
+                "type": "modifier",
+                "variations": [
                     {
-                        "spokenForm": f"{complex_modifiers['last']} <number> <scope>s",
-                        "description": "last <number> instances of <scope> in iteration scope",
+                        "spokenForm": f"{complex_modifiers['ancestor']} <scope>",
+                        "description": "Grandparent containing instance of <scope>",
                     },
-                ),
-            ],
-        },
+                ],
+            }
+        )
+
+    items.append(get_relative_scope(complex_modifiers))
+    items.append(get_ordinal_scope(complex_modifiers))
+
+    return items
+
+
+def get_relative_scope(complex_modifiers: dict[str, str]) -> ListItemDescriptor:
+    variations: list[Variation] = []
+
+    if "previous" in complex_modifiers:
+        variations.append(
+            {
+                "spokenForm": f"{complex_modifiers['previous']} <scope>",
+                "description": "Previous instance of <scope>",
+            }
+        )
+
+    if "next" in complex_modifiers:
+        variations.append(
+            {
+                "spokenForm": f"{complex_modifiers['next']} <scope>",
+                "description": "Next instance of <scope>",
+            }
+        )
+
+    if "previous" in complex_modifiers:
+        variations.append(
+            {
+                "spokenForm": f"<ordinal> {complex_modifiers['previous']} <scope>",
+                "description": "<ordinal> instance of <scope> before target",
+            }
+        )
+
+    if "next" in complex_modifiers:
+        variations.append(
+            {
+                "spokenForm": f"<ordinal> {complex_modifiers['next']} <scope>",
+                "description": "<ordinal> instance of <scope> after target",
+            }
+        )
+
+    if "backward" in complex_modifiers:
+        variations.append(
+            {
+                "spokenForm": f"<scope> {complex_modifiers['backward']}",
+                "description": "single instance of <scope> including target, going backwards",
+            }
+        )
+
+    if "forward" in complex_modifiers:
+        variations.append(
+            {
+                "spokenForm": f"<scope> {complex_modifiers['forward']}",
+                "description": "single instance of <scope> including target, going forwards",
+            }
+        )
```

Comment:
Can we make these shorter per the method done above?

Reply:
done